specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,119 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res9101",
+    "accountName": "sto4445",
+    "api-version": "2020-08-01-preview",
+    "parameters": {
+	 {
+	   "identity": {
+		 "type": "UserAssigned", 
```

Comment:
IT seems the property "type" is not defied in swagger.

Reply:
noted. will verify and fix it. Thanks

Reply:
"type" property is defined in swagger. However, CI is somehow referring to "EncryptionIdentity" element instead of "Identity" element. Will check further.